### ISSUE TITLE
Extract RNA and run qPCR of remaining Lifestage Carryover project samples

### ISSUE THREAD

**Issue by AHuffmyer** [RobertsLab/resources#1891]
We should next extract RNA and run qPCR for the remaining LCO project samples given that we [found differential expression in the seed and spat samples run previously](https://github.com/RobertsLab/resources/issues/1862). In this last round, we analyzed gene expression in conditioned vs unconditioned seed and spat (Oct exposure) under acute stress conditions (Feb exposure). 

There are 80 total samples left. A sheet with the sample information and tube numbers are in a [.csv on GitHub here](https://github.com/RobertsLab/project-gigas-carryover/blob/main/lifestage_carryover/data/sampling_metadata/20240508_rna_extractions.csv). 

These samples would allow us to analyze expression from conditioned vs unconditioned animals (Oct exposure) from adult, spat, seed, and juvenile stages under control or acute stress conditions (Feb exposure). 

There is no strict deadline on this - process as we can. 

@sr320 would you like these samples to perhaps be saved for a summer student project? 

@kubu4 given that we have 80 total samples what is a reasonable timeline to get started? I'd love to learn your extraction and qPCR protocol and help process.

**Comment by kubu4** [comment]
I could start any time. Although, I might need to evaluate kit components and possibly order some new kits... 


Extraction protocol is simply using Zymo kit, so nothing to learn, really; just follow the manual. :)

Reverse transcription and qPCR protocols are described in the Handbook (although, it's possible RT procedure might need some very minor tweaks/updates).

Of course, you're always welcome to check things out in person!

**Comment by AHuffmyer** [comment]
Great! Let me know what you think about inventory and we can check budget availability for kits. As you get started let me know when you are planning to do the work and I'll join for as much of it as I can! :)

**Comment by kubu4** [comment]
Supplies have been ordered.

I'd expect to begin working on this early next week. I'll keep this issue updated as I progress.

**Comment by AHuffmyer** [comment]
Excellent! I'll be on campus on Tuesday the 21st, so I'll check in with you then to see if I can learn/help!

**Comment by AHuffmyer** [comment]
Hey Sam! I'll be on campus on May 30th (Thursday) this week. Let me know if I can do anything to help.

**Comment by kubu4** [comment]
@AHuffmyer - I'm getting started on this today. Can you please remind me where these samples are stored?

**Comment by AHuffmyer** [comment]
Yes! Information is here: https://github.com/RobertsLab/resources/issues/1828.

**Comment by kubu4** [comment]
Alrighty, RNA isolation/quantification is complete.

For qPCRs, you want the same set of genes used in #1862 ?

Also, I'll likely need to order some reagents for reverse transcription and qPCRs. I'll assess situation when I'm in the lab next week.

**Comment by AHuffmyer** [comment]
Yes, lets use the same set of genes since we saw some were differentially expressed and some were not. 

Thank you so much Sam!

**Comment by AHuffmyer** [comment]
What is the current progress for this? No rush at all, just checking in.

**Comment by kubu4** [comment]
Currently lower priority than some other lab stuff I've had to work on.

Will let you know when I resume.

**Comment by kubu4** [comment]
Evaluated consumables/reagents needed.

Will be ~$2700. Will post to Purchases channel in Slack sometime this week to get things ordered.

Hope to start up on this again in the in early Early September.

**Comment by AHuffmyer** [comment]
How is this going @kubu4? Were you able to purchase required supplies? 

Again - no rush, just as you have time. :)

**Comment by kubu4** [comment]
Reagents are in. Just waiting on plates/caps. They've shipped, so hopefully they show up sometime soon...

**Comment by kubu4** [comment]
I'll be diving into this heavily next week! Will share results as they come in...

**Comment by AHuffmyer** [comment]
Thank you!! I'll be on campus Wed so I'm happy to help and/or check in with you then.

**Comment by kubu4** [comment]
Wednesday is the only day I can't/won't be there... Sorry!

**Comment by AHuffmyer** [comment]
All good - thank you!!

**Comment by AHuffmyer** [comment]
Data is looking great Sam! One last question before closing this issue - a few of the seed and spat samples didn't yield RNA as detailed [in this post](https://robertslab.github.io/sams-notebook/posts/2024/2024-03-19-RNA-Isolation---C.gigas-Lifestage-Carryover-Seed-and-Spat/). Do you think there is any benefit in trying RNA isolation with these samples?

**Comment by kubu4** [comment]
Sorry, I think I'm a bit confused. I used the entirety of the sample for RNA isolations. If there are duplicates, I'm certainly happy to isolate RNA from them. Shouldn't be an issue getting usable RNA for other samples any more, as I tweaked the methodology after that round of isolations in March (where some samples failed to yield any RNA).

**Comment by AHuffmyer** [comment]
No problem! I do have some snap frozen samples from these same groups, so I'll think a bit more about whether we should process them. We have n=3 in some groups, so it may be worth processing some of the snap frozen samples. I'll close this issue now and open a new one if we need to use the frozen samples. Thank you!!
